src/components/RsvpResponseContent.js

```diff
@@ -44,11 +75,30 @@ const RsvpResponseContent = ({ items, activeValue, limit = false }) => {
 					role="tabpanel"
 					aria-labelledby={`gp-rsvp-${value}-tab`}
 				>
-					<RsvpResponseCard
-						value={value}
-						limit={limit}
-						responses={rsvpResponse}
-					/>
+					{!editMode && (
+						<RsvpResponseCard
+							value={value}
+							limit={limit}
+							responses={rsvpResponse}
+						/>
+					)}
+					{editMode && (
+						<div>
+							<Autocomplete completers={autocompleters}>
+								{({ listBoxId, activeId, onChange }) => (
+									<div>
+										<input
+											onChange={onChange}
+											aria-controls={listBoxId}
+											aria-activedescendant={activeId}
+										/>
+
+										<ul id={listBoxId}></ul>
+									</div>
+								)}
+							</Autocomplete>
```

Comment:
This is how we are supposed to implement this component. However, I am stuck on a strange error.
(error happens both on the block editor and the front end).
<img width="1365" alt="Screenshot 2024-01-21 at 10 10 35 AM" src="https://github.com/GatherPress/gatherpress/assets/223919/8b0ae8f0-beec-48cd-af71-f34dbababa5c">

Reply:
@jmarx this should only exist on the backend (not the frontend). There's examples of other things in GatherPress using Autocomplete, so would take a look at those as well. Thx!